src/usercontent/index.ts

```diff
@@ -23,32 +24,29 @@ function remoteRender(event) {
 
     // Apply image style after so we can steal the anchor's colour.
     // Style copied from a rendered version of mx_MFileBody_download_icon
-    img.style = (data.imgStyle || "" +
-        "width: 12px; height: 12px;" +
-        "-webkit-mask-size: 12px;" +
-        "mask-size: 12px;" +
-        "-webkit-mask-position: center;" +
-        "mask-position: center;" +
-        "-webkit-mask-repeat: no-repeat;" +
-        "mask-repeat: no-repeat;" +
-        "display: inline-block;") + "" +
-
-        // Always add these styles
-        `-webkit-mask-image: url('${data.imgSrc}');` +
-        `mask-image: url('${data.imgSrc}');` +
-        `background-color: ${a.style.color};`;
+    // @ts-ignore
+    img.style = data.imgStyle ?? "";
+    img.style.width = "12px";
+    img.style.height = "12px";
+    img.style.webkitMaskSize = "12px";
+    img.style.webkitMaskPosition = "center";
+    img.style.webkitMaskRepeat = "no-repeat";
+    img.style.display = "inline-block";
+    img.style.webkitMaskImage = `url('${data.imgSrc}')`;
+    img.style.backgroundColor = `${a.style.color}`;
```

Comment:
This doesn't look equivalent - previously it was `data.imgStyle || ("width: 12px; height:12px; -webkit-mask-size: 12px; ...")` because of how string concatenation works.

Reply:
Oops, misread that line :sweat_smile: